deploy/podman/flightctl-db/flightctl-db-config/enable-superuser.sh

```
@@ -19,18 +19,113 @@ echo "PostgreSQL is ready, configuring permissions..."
 # Ensure POSTGRESQL_MASTER_USER is treated as a superuser
 if [ -n "${POSTGRESQL_MASTER_USER}" ]; then
     echo "Granting superuser privileges to ${POSTGRESQL_MASTER_USER}"
-    
+
     # Check if user exists, create if it doesn't
     if ! _psql -U postgres -tAc "SELECT 1 FROM pg_roles WHERE rolname = '${POSTGRESQL_MASTER_USER}'" | grep -q 1; then
         echo "Creating master user ${POSTGRESQL_MASTER_USER}"
-        _psql -U postgres -c "CREATE ROLE \"${POSTGRESQL_MASTER_USER}\" WITH LOGIN PASSWORD '${POSTGRESQL_MASTER_PASSWORD}';"
+        _psql -U postgres -c $'DO $$BEGIN
+            IF NOT EXISTS (SELECT 1 FROM pg_roles WHERE rolname = quote_ident(''${POSTGRESQL_MASTER_USER}'')) THEN
```

Shellcheck (0.10.0)

[warning] Did you forget to close this single quoted string?

(SC1078)